packages/sshnoports/bundles/shell/install.sh

```diff
@@ -0,0 +1,296 @@
+#!/bin/sh
+
```

Comment:
New POSIX sh compliant install script, should also work on ash!

- I make some assumptions, remove some smoke and mirrors, which has resulted in 4x less shell script to maintain, and the shell script is also much less complicated as well